charts/datahub/subcharts/datahub-gms/templates/deployment.yaml

```diff
@@ -94,12 +94,17 @@ spec:
             - name: DATAHUB_ANALYTICS_ENABLED
               value: "{{ .Values.global.datahub_analytics_enabled }}"
             - name: EBEAN_DATASOURCE_USERNAME
-              value: "{{ .Values.global.sql.datasource.username }}"
+              value: "{{ .Values.sql.datasource.username | default .Values.global.sql.datasource.username }}"
             - name: EBEAN_DATASOURCE_PASSWORD
+              {{- $passwordValue := .Values.sql.datasource.password.value | default .Values.global.sql.datasource.password.value }}
+              {{- if $passwordValue }}
```

Comment:
again, the value of a password will only be used if it has been set, otherwise it will attempt to use a k8s secret